src/routes/user.route.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const authController = require('../controllers/auth.controller.js');
+const { authMiddleware } = require('../middlewares/authMiddleware.js');
+const catchError = require('../utils/catchError.js');
+
+const userRouter = new express.Router();
+
+userRouter.get('/', authMiddleware, catchError(authController.activate));
```

Comment:
The route uses `authController.activate`, which seems incorrect for a user route. Consider using a function from `userController`, such as `getAllActivated`, to handle this route.